zerver/lib/actions.py

```diff
@@ -3751,8 +3751,13 @@ def do_change_stream_description(stream: Stream, new_description: str) -> None:
 
 def do_create_realm(string_id: str, name: str,
                     emails_restricted_to_domains: Optional[bool]=None) -> Realm:
+    from zerver.lib.server_initialization import create_internal_realm, server_initialized
```

Comment:
I think we should do a local in-function import in `server_initialization.py` instead, to compartmentalize the unclean code there rather than here.

Reply:
Also, I guess we should actually be doing `Realm.objects.exists()` in `server_initialization.py`, not getting the count, as a very minor optimization?

Reply:
Good points, pushed again with both fixes.